src/app/date-picker/date-picker-config.model.ts

```diff
@@ -8,6 +8,8 @@ export interface IDatePickerConfig extends IDayCalendarConfig,
                                            ITimeSelectConfig {
   closeOnSelect?: boolean;
   closeOnSelectDelay?: number;
+  openOnFocus?: boolean;
```

Comment:
this should be available in all type of pickers, not only timeSelect

Reply:
This works for all pickers

Reply:
But it is odd that it is present only in the timeSelect

Reply:
I don't understand what do you mean? This configuration as far as I am concerned is shared among other pickers, I have checked the demo page, and it worked perfectly.